lib/__tests__/navigation.test.js

```diff
@@ -54,6 +53,20 @@ describe('Navigation metalsmith plugin', () => {
         stats: {},
         filepath: 'test/test-pattern-with-custom-text/index.njk',
         menuText: 'Custom text!'
+      },
+      'test/test-pattern/with-additional-segment/index.html': {
+        contents: Buffer.from('test pattern with additional directory segment'),
+        mode: '0644',
+        stats: {},
+        filepath: 'test/test-pattern/with-additional-segment/index.njk',
+        status: 'development'
```

Comment:
If this should be filtered out regardless of the status it would be good to remove this.

Reply:
ooo - well spotted. C&P fail